src/benchmark/join_order_benchmark.cpp

```diff
@@ -113,5 +114,27 @@ int main(int argc, char* argv[]) {
   auto query_generator =
       std::make_unique<FileBasedQueryGenerator>(*benchmark_config, query_path, non_query_file_names, query_subset);
 
-  BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context}.run();
+  auto benchmark_runner =
+      BenchmarkRunner{*benchmark_config, std::move(query_generator), std::move(table_generator), context};
+
+  std::cout << "- Adding indexes to SQLite... " << std::flush;
+
+  if (benchmark_config->verify) {
+    // Add indexes to SQLite. This is a hack until we support CREATE INDEX ourselves and pass that on to SQLite
+    for (const auto& table_name : StorageManager::get().table_names()) {
+      // SQLite does not support adding primary keys, so we just add a regular index
+      benchmark_runner.sqlite_wrapper->raw_execute_query(std::string{"CREATE INDEX "} + table_name + "_primary ON " +  // NOLINT
+                                                         table_name + "(id)");
+    }
+
+    std::ifstream foreign_key_file(std::string{DEFAULT_QUERY_PATH} + "/fkindexes.sql");  // NOLINT
+    std::string foreign_key_definition;
+    while (getline(foreign_key_file, foreign_key_definition)) {
+      benchmark_runner.sqlite_wrapper->raw_execute_query(foreign_key_definition);
+    }
+  }
+
+  std::cout << "done." << std::endl;
```

Comment:
You could consider adding timing info (i.e. how long creating the indexes took) here. Most other setup steps in the benchmarks do the same